libindi/drivers/telescope/ieqprodriver.cpp

```diff
@@ -910,8 +910,8 @@ bool set_ieqpro_guide_rate(int fd, double rate)
     int nbytes_read    = 0;
     int nbytes_written = 0;
 
-    int num = rate * 1000;
-    snprintf(cmd, 16, ":RG%03d#", num);
+    int num = rate * 100;//zy2018 set RA and Dec with the same value
+    snprintf(cmd, 16, ":RG%02d%02d#", num, num);//zy2018
```

Comment:
he documentation here: http://www.ioptron.com/v/ASCOM/RS-232_Command_Language2014_V2.0.pdf

specifies this format:

Command: “:RGnnn#”
Response: “1”
Sets the guiding rate n.nn * sidereal rate. nnn can only be entered in the range of [10, 90].